src/containers/my-cooperations/cooperation-notes/note-view/NoteView.styles.tsx

```diff
@@ -0,0 +1,59 @@
+import palette from '~/styles/app-theme/app.pallete'
+import { TypographyVariantEnum } from '~/types'
+
+export const styles = {
+  container: (currentUser: boolean, isPrivate: boolean) => ({
+    maxWidth: '400px',
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '16px',
+    p: '20px',
+    mt: '24px',
+    borderRadius: '2px 2px 6px 6px',
+    borderTop: `4px solid ${
+      isPrivate ? palette.basic.blueGray : palette.basic.turquoise
```

Comment:
maybe better to use borderColor:  isPrivate ? basic.blueGray : basic.turquoise?

Reply:
used borderColor